commands/provider/wallet_provider.go

```diff
@@ -96,9 +96,9 @@ func (p *WalletProvider) call(task string, inputs interface{}, output interface{
 }
 
 func (p *WalletProvider) parseResult(r *coreapi.ResultData, output interface{}) error {
-	if r.OutputKey == "error" {
+	if r.Error != "error" {
```

Comment:
```suggestion
	if r.Error != "" {
```
missing modif here

Reply:
@krhubert why this comment is solved?

Reply:
Because i forced pusehd when I changed it

Reply:
I will fix it in 11th step. so now we won't trigger again tests and builds for 2 branches